lib/config-validator.ts

```diff
@@ -54,19 +60,19 @@ type PackageJson = {
       try {
         if (!(await pathExists(file))) {
           returnVal = 1;
-          logger.error(`${file} does not exist`);
+          logger.error({ file }, 'File does not exist');
           break;
         }
         const parsedContent = await getParsedContent(file);
         try {
           logger.info(`Validating ${file}`);
           await validate(file, parsedContent);
         } catch (err) {
-          logger.warn({ err }, `${file} is not valid Renovate config`);
+          logger.warn({ file, err }, 'File is not valid Renovate config');
```

Comment:
the problem with metadata is that's not visible on dependency dashboard. there only the message is visible. we should define some known metadata which we show there too, like fileName, err.message

Reply:
Is this particular file used in DD though?

Reply:
This particular error should matter since this informs that the file which contain Renovate config is valid. Users already know the file in question.

The part about defining a known metadata can be done in a follow-up PR.
Is this the logic where we want changes?
https://github.com/renovatebot/renovate/blob/43fa615e05ef5b95dbf0e93b19e37baaae6649e9/lib/workers/repository/dependency-dashboard.ts#L157-L169

Reply:
ok